### PR TITLE
feat(colors): change colors for profiles & remove old api stats

### PR DESCRIPTION
**Pull request by ugcodrr**
feat(colors): change colors for profiles as well as remove old api stats

### CODE REVIEW

File: apps/discord-bot/src/commands/blitzsg/blitzsg.profile.tsx [hunk 1]
```diff
@@ -111,7 +111,7 @@ export const BlitzSGProfile = ({
       let level = stats.prestige ? `§6${'✫'.repeat(stats.prestige)}` : romanNumeral(stats.level);
       if (stats.level === 10) level = `§l${level}`;
       sidebar.push([t('stats.level'), level, colors[stats.level - 1]]);
-
+      sidebar.push([t('stats.exp'), t(stats.exp), '§b']);
```
Comment (jacobk999) [inline apps/discord-bot/src/commands/blitzsg/blitzsg.profile.tsx:114]: REMOVE THIS. ITS SO SHIT WTF

File: packages/assets/src/index.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { join } from 'path';
 import { Image, loadImage } from 'skia-canvas';
 
 const PATH = '../../assets';
-const PRIVATE_PATH = join(PATH, 'private', 'package.json');
+const PRIVATE_PATH = join(PATH, 'private');
 
 const hasPrivateAssets = existsSync(PRIVATE_PATH);
```
Comment (jacobk999) [inline packages/assets/src/index.ts:8]:
```suggestion
const hasPrivateAssets = existsSync(PRIVATE_PATH, 'package.json');
```

File: packages/schemas/src/player/gamemodes/murdermystery/mode.ts [hunk 3]
```diff
@@ -23,7 +20,6 @@ export class MurderMysteryMode {
     this.wins = data[`wins${mode}`];
     this.gamesPlayed = data[`games${mode}`];
     this.kills = data[`kills${mode}`];
-    this.deaths = data[`deaths${mode}`];
-    this.kdr = ratio(this.kills, this.deaths);
+    this.kdr = ratio(this.kills, data[`kills${mode}`]);
```
Comment (jacobk999) [inline packages/schemas/src/player/gamemodes/murdermystery/mode.ts:23]:
```suggestion
    this.kdr = ratio(this.kills, data[`deaths${mode}`]);
```

File: packages/schemas/src/player/gamemodes/uhc/mode.ts [hunk 4]
```diff
@@ -21,7 +21,7 @@ export class UHCMode {
   @Field()
```
Comment (jacobk999) [inline packages/schemas/src/player/gamemodes/uhc/mode.ts:21]:
```suggestion
  @Field({ leaderboard: { enabled: false }})
```

**Review by jacobk999 — CHANGES_REQUESTED** [review]

File: apps/discord-bot/src/commands/blitzsg/blitzsg.profile.tsx (hunk 1 above)
Comment (ugcodrr) [reply apps/discord-bot/src/commands/blitzsg/blitzsg.profile.tsx:114]: this aged perfectly

**Review by jacobk999 — APPROVED** [review]

**Pull request merged**